tests/data_context/test_data_context_config_variables.py

```diff
@@ -675,16 +675,21 @@ def test_create_data_context_and_config_vars_in_code(tmp_path_factory, monkeypat
     monkeypatch.setenv("DB_HOST_FROM_ENV_VAR", "DB_HOST_FROM_ENV_VAR_VALUE")
 
     datasource_config = DatasourceConfig(
-        class_name="SqlAlchemyDatasource",
-        credentials={
-            "drivername": "postgresql",
-            "host": "$DB_HOST",
-            "port": "65432",
-            "database": "${DB_NAME}",
-            "username": "${DB_USER}",
-            "password": "${DB_PWD}",
+        class_name="Datasource",
```

Comment:
Using new style datasource since this test is still relevant.

Reply:
🚀